tfe_hardware/qcodes_instrument_drivers/Sinko/AD5760.py

```diff
@@ -0,0 +1,170 @@
+'''
+A simple driver to control the AD5760 current source
+'''
+
+__author__ = "Kaushik Singirikonda"
+__email__ = 'ks105@illinois.edu'
+
+from functools import partial
+from typing import Optional, Union, Any
+import time,sys
+import clr  # install pythonnet
+
+from qcodes.instrument.parameter import DelegateParameter
+from qcodes import (Instrument, validators as vals)
+from qcodes.instrument.channel import InstrumentChannel
+from qcodes.utils.validators import Numbers, Bool, Enum, Ints
+
+'''
+To use this current source, you first have to install and setup the ACE software. Upon connecting to the AD5760
+board with your computer using the software, you have to initialize a setup on python to talk to the software to
+enable writing.
+
+Following this, you can simply run this code to operate a stable current source.
+
+
+'''
+
+
+class AD5760(Instrument):
+
+    def __init__(self, name: str, host_name: str, path: str, syspath: str, terminator: str = "\n",
```

Comment:
is syspath the string to a file that it needs to load? what is the difference between path and syspath?

Reply:
path is essentially how we talk to the device (via the software) using python. Syspath is the parent folder of the software

Essentially, path adds onto the syspath... but we need it separated and as a variable. Whereas syspath is constant for a PC,, you just need to change it if you are using the driver on a new device.

Reply:
Could you add a documentation specifying what each is and how they differentiate from each other?

Reply:
Alright, I can do that... now that I think of it, its not as trivial as I had assumed it to be

Reply:
yeah, it usually is the case. When writing the documentation for a driver think of someone a few years in the future that knows absolutely nothing on how this thing works. The documentation ideally should be good enough for that person to figure out how to use this piece of software on its own